src/main/java/org/elasticsearch/index/translog/TranslogStats.java

```diff
@@ -34,13 +34,12 @@
 public class TranslogStats implements ToXContent, Streamable {
 
     private long translogSizeInBytes = 0;
-    private int estimatedNumberOfOperations = 0;
+    private int estimatedNumberOfOperations = -1;
```

Comment:
I wanted to distinguish between cases we don't know (like when opening a translog file for the first time) and an actual 0 . I think it's important.